Anlab.Core/Domain/Order.cs

```diff
@@ -69,5 +70,22 @@ public void SaveDetails(OrderDetails details)
             JsonDetails = JsonConvert.SerializeObject(details);
         }
 
+        public IList<TestItemModel> GetTestDetails()
+        {
+            try
+            {
+                return JsonConvert.DeserializeObject<IList<TestItemModel>>(SavedTestDetails);
+            }
+            catch (JsonSerializationException)
```

Comment:
@laholstege this catch does nothing except eat the stack trace, so just remove the whole try/catch